style.css

```diff
@@ -0,0 +1,68 @@
+.header-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 1px;
+  background: #fff;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text1 {
+  width: 327px;
+  height: 146px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.text2 {
+  width: 117px;
+  height: 24px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.main-section {
```

Comment:
Unfortunately, the background image is still not exist. It is because there is no image file in your folders.
<img width="511" alt="Screenshot 2023-01-19 at 8 12 39 PM" src="https://user-images.githubusercontent.com/25887253/213513876-dcf9a33d-ff4a-4929-ad99-d84414387de9.png">
<img width="425" alt="Screenshot 2023-01-19 at 8 14 30 PM" src="https://user-images.githubusercontent.com/25887253/213514007-62d99f6a-b1a1-49e9-a532-25f5ba8c79bb.png">